v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
@bosonogi Question about honoring requirements is a different test. This one tests if the value will be propagated to the command line.